File: 2015/gara4/remix/att/remix.cpp
#include <stdio.h>
#include <assert.h>

#define MAXN 100000

void ripulisci(int N, char remix[], char testo[]) {
    // Mettete qui il codice della soluzione
}


char remix[MAXN+1], testo[MAXN+1]; // Maggiore di MAXN per contenere newline

int main() {
    FILE *fr, *fw;
    int N;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(1 == fscanf(fr, "%d\n", &N));
    assert(remix == fgets(remix, N + 1, fr));

    ripulisci(N, remix, testo);
    fprintf(fw, "%s\n", testo);
    fclose(fr);
    fclose(fw);
    return 0;
}

File: ed1/gara4/remix/att/remix.pas
procedure ripulisci(N: longint; var remix, testo: array of char);
begin
    (* Mettete qui il codice della soluzione *)
end;


const
    MAXN = 100000;
var
    N               : longint;
    remix, testo    : array[0..MAXN] of char;
    fr, fw          : text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N);
    readln(fr, remix);
    ripulisci(N, remix, testo);
    writeln(fw, testo);
    close(fr);
    close(fw);
end.
